react-app-drawer-component/src/app-drawer.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+export default class AppDrawer extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      open: false,
+      list: this.props.list
+    };
+
+    this.MenuList = this.MenuList.bind(this);
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  MenuList(props) {
+    const list = props.list;
+    const listItems = list.map(item => {
+      return (
+        <li className='list-item' key={item.id}>{item.name}</li>
+      );
+    });
+    return (listItems);
+  }
+
+  handleClick(event) {
+
+    if (event.target.className === 'fa-solid fa-bars') {
+      this.setState({
+        open: !this.state.open
+      });
+    } else if (event.target.className === 'darken') {
+      this.setState({
+        open: !this.state.open
+      });
+    }
```

Comment:
Since the same action is being performed in both `if` statements you can use a single if with an `or (`||`).
```JAVASCRIPT
const { className } = event.target;

if (className === 'fa-solid fa-bars' || className === 'darken') {
  this.setState({
    open: !this.state.open
  });
}
```

Reply:
true!